storage/innobase/row/row0log.cc

```diff
@@ -2037,6 +2037,7 @@ row_log_table_apply_update(
 
 		When applying the subsequent ROW_T_DELETE, no matching
 		record will be found. */
+		return(DB_SUCCESS);
```

Comment:
This changes the behavior of the function, are you sure?
Also space before (

Reply:
No, I messed up the conflict resolution somehow here - I fixed this locally and will push soon.